### DISCUSSION TITLE
Horizontal scrollbar

### DISCUSSION THREAD

**Discussion by mahmoodjm** [KingSora/OverlayScrollbars#405] [Q&A] [answered]
Hi, 
I have a div which only scrolls horizontally(exactly what I need). 
Right now if I want to scroll the content with the mouse I have to use shift + the scroll wheel to do so. However, I want to use the regular scroll wheel without the shift to scroll horizontally, is this possible?

Similar to these two examples from another plugin(custom-scrollbar):
![scrollbar](https://user-images.githubusercontent.com/7316226/144862250-97fba771-1925-4f88-a0d5-24cfbd0f94f2.jpg)

http://manos.malihu.gr/repository/custom-scrollbar/demo/examples/complete_examples.html

**Comment by KingSora** [comment, marked as answer]
Hello :)

I've answered a similar question here: https://github.com/KingSora/OverlayScrollbars/issues/211 

The example isn't perfect, but it does what it should

**Reply by mahmoodjm** [reply]
Thank you so much.
